README.md

```diff
@@ -36,6 +36,10 @@ review the Example Flows and JSON Schema for the language.
 | [JSON Schema](/stix/attack-flow-schema-2.0.0.json)                                   | The language specification expressed as a JSON Schema.                                      |
 | [Example Flows](/corpus/)                                                                  | A corpus of example Attack Flows.                                        |
 
+### Run Attack Flow using Docker Compose
```

Comment:
These deployment instructions need to go into the docs/builder.rst file under the existing Docker heading.

Reply:
done